ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js

```diff
@@ -543,21 +540,21 @@ export default class ConfirmApproveContent extends Component {
           display={DISPLAY.FLEX}
           className="confirm-approve-content__icon-display-content"
         >
-          <Box display={DISPLAY.FLEX}>
+          <Box display={DISPLAY.FLEX} alignItems={AlignItems.center}>
             <UrlIcon
               className="confirm-approve-content__siteimage-identicon"
               fallbackClassName="confirm-approve-content__siteimage-identicon"
               name={origin}
               url={siteImage}
             />
-            <Typography
-              variant={TypographyVariant.H6}
-              fontWeight={FONT_WEIGHT.NORMAL}
+            <Text
+              variant={TextVariant.bodySm}
+              as="h6"
               color={TextColor.textAlternative}
-              boxProps={{ marginLeft: 1, marginTop: 2 }}
+              marginLeft={1}
```

Comment:
Hey @spiritanand, I just added a little alignment fix here so we can get this through! cc @garrettbear 

<img width="603" alt="Screenshot 2023-03-10 at 10 10 24 AM" src="https://user-images.githubusercontent.com/8112138/224391904-645bfae3-a4c7-4428-bbd6-02bbda9dfca7.png">
<img width="617" alt="Screenshot 2023-03-10 at 10 09 51 AM" src="https://user-images.githubusercontent.com/8112138/224391927-8a74fb5e-2e86-43ec-9c4b-c1062269c34b.png">